include/LmmsTypes.h

```diff
@@ -55,20 +51,6 @@ using mix_ch_t = uint16_t;		// Mixer-channel (0 to MAX_CHANNEL)
 using jo_id_t = uint32_t; // (unique) ID of a journalling object
 
 
-constexpr ch_cnt_t DEFAULT_CHANNELS = 2;
```

Comment:
I know you copied this name from elsewhere, but shouldn't constants like this normally be in capitals?